### DISCUSSION TITLE
How to add sequence base setup times to flexible job shop problem?

### DISCUSSION THREAD

**Discussion by ensarovski** [google/or-tools#3158] [CP-SAT questions] [answered]
I am new to Python and I like your project very much, I need to add setup times during job change on a machine and I cannot add it, can you help?
**https://github.com/google/or-tools/blob/stable/examples/python/flexible_job_shop_sat.py** this is the code that I used for the problem.

**Comment by lperron** [comment, marked as answer]
Answered on or stackexchange.
